release/cli/pkg/images/images.go

```diff
@@ -116,6 +116,19 @@ func CopyToDestination(sourceAuthConfig, releaseAuthConfig *docker.AuthConfigura
 
 		return nil
 	})
+	// Sign public ECR image using AWS signer and notation CLI
+	// notation sign public.ecr.aws/y8b4r0e8/eksa-controller:latest8 @sha256:ff76a27fb06d711dafd399b97b142a23217ab171eb0468aeaa9374b69849a1e0
+	// --plugin com.amazonaws.signer.notation.plugin --id arn:aws:signer:us-east-1:189183948571:/signing-profiles/notation_test_nov_8
+	err = retrier.Retry(func() error {
+		cmd := exec.Command("notation", "sign", releaseImageUri, "--plugin", "com.amazonaws.signer.notation.plugin", "--id", awsSignerProfileArn)
+		out, err := commandutils.ExecCommand(cmd)
+		fmt.Println(out)
+		if err != nil {
+			return fmt.Errorf("executing skopeo copy command: %v", err)
+		}
+
+		return nil
+	})
 	if err != nil {
```

Comment:
Let's move this check above after the skopeo copy retry, and do a separate error check for the notation retry.